index.js

```diff
@@ -4,31 +4,41 @@ const {Client} = require('discord.js');
 const Bot = require('./src/Bot.js');
 
 const client = new Client();
-const bot = new Bot({}, 10);
+const bot = new Bot(10);
 
 client.on('ready', () => {
   console.log('READY');
 });
 
 client.on('message', message => {
-  bot.notify(message.channel.id, new Date());
+  const {channel} = message;
```

Comment:
Channel or message? These are different thing, right? Or am I confused?

Reply:
Sorry, not sure what you mean?

Reply:
Oh, `const {property} = parent` is object destructuring, sets `property` to `parent.property`

Reply:
I'm a bit confused by the `{channel} = message` part.. Tho maybe I'm missing some syntax that's new to me; what do the `{` and `}` mean in this context?

Reply:
If it's syntax sugar for `const channel = message.channel` then I've said nothing :roll_eyes:

Reply:
Yeah, exactly